client/src/components/ReviewsAddForm/index.jsx

```diff
@@ -1,11 +1,68 @@
 import React from 'react';
 import styles from './ReviewsAddForm.css';
 
-// Will need to have an onClick event listener and handler
-const ReviewsAddForm = () => (
-  <div className={styles.reviewsAddForm}>
-    <button className={styles.addButton} type="submit">Add Review</button>
-  </div>
-);
+class ReviewsAddForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      overallRating: '',
+      email: '',
+      reviewUsername: '',
+      reviewSummary: '',
+      reviewBody: '',
+      recommendProduct: '',
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  handleSubmit(event) {
+    event.preventDefault();
+    // Need to have function that adds reviews to a tile and adds to list - function passed as props
+    // this.addReview(this.state.______);
+    // Need to reset state);
+  }
+
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.handleSubmit}>
+          <h2>Write New Review</h2>
+          <h4>Overall Rating:</h4>
+          <input onChange={this.handleChange} type="text" name="overallRating" value={this.state.overallRating} placeholder="Rating: 1 - 5" />
```

Comment:
[MDN inputs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input)

You can add built-in validators to your inputs so that you dont have to worry about people entering bad data. For example, you can use maxlength, min, and max to constrain your input to accept 1 character input, that has to be a number between 1 and 5

Reply:
oh but i just realized this is a placeholder and you're prob going to replace this with a Star Rating type thing, or something where you can constrain the ratings down to 1/4 stars. carry on :P

Reply:
Great catch! I will work on adding the validators next (since that is indeed one of the requirements). Indeed most of these items are placeholders. For this PR, currently trying to build out the skeleton and submitting PR so it doesn't get too long.